cmd/otel-allocator/allocation/strategy.go

```diff
@@ -149,7 +149,7 @@ func init() {
 		panic(err)
 	}
 	err = Register(perNodeStrategyName, func(log logr.Logger, opts ...AllocationOption) Allocator {
-		return newAllocator(log, newPerNodeStrategy(), opts...)
+		return newAllocator(log, newPerNodeStrategy(newleastWeightedStrategy()), opts...)
```

Comment:
I would much prefer the fallback strategy be `consistent-hashing`, and that should also be the default if we make this configurable.

The reason for this is that the `least-weighted` strategy isn't stable over time - the end result depends on what order changes to the set of targets arrive. To put it another way, with `least-weighted`, if you start multiple target allocator Pods, there's no guarantee that they give the same target allocation unless they're started at the exact same time. In comparison, `consistent-hashing` is stable, and you can scale the Target Allocator with it.

Reply:
I changed the strategy to consistent-hashing